mbed_tools_ci/utils/git_helpers.py

```diff
@@ -439,3 +439,14 @@ def _get_remote(self) -> Optional[Any]:
         except (IndexError, ValueError) as e:
             logger.warning(e)
             return None
+
+    def get_changes_against_master(self) -> List[str]:
+        """Returns a list of files changed against master branch."""
+        master_branch_commit = self.repo.commit('master')
+        current_branch_commit = self.repo.commit(self.get_current_branch())
+        changes = self.get_changes_list(
+            self.get_branch_point(master_branch_commit, current_branch_commit),
+            current_branch_commit,
+            change_type='a'
```

Comment:
this would list only `added` files. Could we either mention this in the function name or let it as an argument of the method

Reply:
Good point, I'll change the name.